_tips/designing.ja.md

```diff
@@ -0,0 +1,669 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:"
+
+title: "ウェブ・アクセシビリティのためのデザイン – 始めるにあたってのヒント"
+title_html: "ウェブ・アクセシビリティのためのデザイン"
+nav_title: デザインのためのヒント
+lang: ja  # Change "en" to the translated-language shortcode
+last_updated: 2024-03-29   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+resource:
+  ref: /tips/  # Do not change this
+
+navigation:
+  previous: /tips/writing/ # Do not change this
+  next: /tips/developing/ # Do not change this
+
+# translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
```

Comment:
@jaykishigami Please remove the hash sign and the space from the beginning of this line.